draft-ietf-tls-tls13.md

```diff
@@ -5560,17 +5563,21 @@ ClientHello without 0-RTT data.
 To avoid this error condition, multi-server deployments SHOULD ensure a uniform
 and stable deployment of TLS 1.3 without 0-RTT prior to enabling 0-RTT.
 
-## Middlebox Compatibility Mode {#middlebox}
+## Network Intermediary Compatibility Mode {#intermediary}
 
-Field measurements have found that a significant number of middleboxes
-misbehave when a TLS client/server pair negotiates TLS 1.3. Implementations
+Field measurements have found that a significant number of network
+intermediaries, or "middleboxes,"
+can cause the TLS endpoints to fail to negotiate TLS 1.3.
```

Comment:
Independent to this PR, but related: is it possible to provide a reference for these "field measurements"? Something like https://www.ietf.org/mail-archive/web/tls/current/msg24517.html (but preferably something more specific). Might be useful for the archeologists in the future.

Reply:
Perhaps bring that up on the list.  A non-normative link to @davidben's IETF-100 slides might be do-able.